tests/2019-05-01/Microsoft.NetApp.tests.json

```diff
@@ -0,0 +1,506 @@
+{
+  "tests": [

```

Comment:
>tests [](start = 3, length = 5)

is this a merge conflict?

Reply:
Not sure why it does that, diff against master shows only kusto's schema as changed. Anyway openined another PR, please take a look: https://github.com/Azure/azure-resource-manager-schemas/pull/617